modules/common-non-darwin.nix

```diff
@@ -0,0 +1,114 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# configuration options that do not exist for nix-darwin
+{ lib, pkgs, config, options, inputs, ... }:
+{
+  imports = [
+    ./services/nginx.nix
+  ];
+
+  options.services.nginx.addSecurityHeaders = lib.mkOption {
+    description = "whether to add security headers to all nginx virtualHosts";
+    type = lib.types.bool;
+    default = true;
+  };
+
+  config = {
+
+    # Default to /tmp on disk, move it to tmpfs if server big enough
+    boot.cleanTmpDir = true;
+    documentation.nixos.enable = false;
+
+    programs.mosh.enable = true;
+
+    programs.atop.enable = false;
+
+    security.acme = {
+      defaults.email = "operations@serokell.io";
+      acceptTerms = true;
+    };
+
+    services.earlyoom.enable = lib.mkDefault true;
+    services.mysql.package = lib.mkOptionDefault pkgs.mariadb;
+    services.postgresql.package = lib.mkOptionDefault pkgs.postgresql_12;
+
+    security.sudo.wheelNeedsPassword = false;
+
+    services.openssh = {
+      enable = true;
+      passwordAuthentication = false;
+      ports = [ 17788 ];
+    };
+
+    users.mutableUsers = false;
+
+    services.prometheus.exporters.process = {
+      enable = true;
+      settings.process_names = [
+        {
+          name = "{{ .Matches.Wrapped }}";
+          cmdline = [ "^/nix/store[^ ]*/(?P<Wrapped>[^ /]*)" ];
+        }
+        {
+          name = "{{ .Matches.Command }}";
+          cmdline = [ "(?P<Command>[^ ]+)" ];
+        }
+      ];
+    };
+
+    services.prometheus.exporters.node = {
```

Comment:
TIL, there is https://github.com/prometheus/node_exporter/blob/master/examples/launchctl/README.md

Reply:
Though, firewall still has to be configured manually for this :disappointed: